args/src/lib.rs

```diff
@@ -1034,6 +1034,14 @@ Note: multiple interfaces can be specified separated by commas and no spaces"
     )]
     metrics_address: SocketAddr,
 
+    /// Pyroscope server address for profiling uploads
+    #[arg(
+        long,
+        value_name = "ALLOY_ADDR:PORT",
```

Comment:
The `value_name` parameter says "ALLOY_ADDR:PORT" but this should actually be a URL (with scheme like `http://`). Consider changing to something like "PYROSCOPE_URL" to better reflect that it expects a full URL, not just an address:port.

```suggestion
        value_name = "PYROSCOPE_URL",
```